source/go-perun/index.rst

```diff
@@ -0,0 +1,49 @@
+Go-Perun
+========
+
+`go-perun <https://github.com/perun-network/go-perun>`_ is a Go implementation of the Perun state channel protocols.
```

Comment:
```suggestion
`go-perun <https://github.com/hyperledger-labs/go-perun>`_ is a Go implementation of the Perun state channel protocols.
```

Reply:
We did not review the links in all our projects yet to point to `HLL` instead of `perun-network`. This discussion is a good reminder.